packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -54,6 +54,13 @@ export interface DeploymentCircuitBreaker {
    */
   readonly rollback?: boolean;
 
+  /**
+   * Whether to use the explicit DeploymentControllerType.ECS when defining the circuit breaker.
+   * To avoid a service replacement when add a CircuitBreaker to an existing ECS service, set this to False.
+   * @default true
```

Comment:
Would it not be better to set this parameter to false as a default? 
This would prevent service replacements from happening by accident and should not have any drawbacks since Cloudformation doesnt need the explicit Deploymentcontrollertype set to ECS right?

Reply:
I believe https://github.com/aws/aws-cdk/pull/22467#discussion_r998894146 was a reply to this